1970-01-01T00:33:41+00:00	docker.logs	{"log":"W1201 11:37:25.169769       1 watcher.go:220] watch chan error: etcdserver: mvcc: required revision has been compacted\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"time=\"2021-12-01T11:37:28Z\" level=error msg=\"Port 30000 for service reactapp-service is already opened by another service\"\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"W1201 11:37:29.204040       1 watcher.go:220] watch chan error: etcdserver: mvcc: required revision has been compacted\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:37:29.594078 I | etcdserver/api/etcdhttp: /health OK (status code 200)\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"W1201 11:37:36.673127       1 watcher.go:220] watch chan error: etcdserver: mvcc: required revision has been compacted\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"W1201 11:37:37.484875       1 watcher.go:220] watch chan error: etcdserver: mvcc: required revision has been compacted\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:37:39.598905 I | etcdserver/api/etcdhttp: /health OK (status code 200)\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"I1201 11:37:41.435263       1 client.go:360] parsed scheme: \"passthrough\"\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"I1201 11:37:41.435623       1 passthrough.go:48] ccResolverWrapper: sending update to cc: {[{https://127.0.0.1:2379  <nil> 0 <nil>}] <nil> <nil>}\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"I1201 11:37:41.435900       1 clientconn.go:948] ClientConn switching balancer to \"pick_first\"\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:37:41.440230 I | embed: rejected connection from \"127.0.0.1:59904\" (error \"open /run/config/pki/etcd/server.crt: no such file or directory\", ServerName \"\")\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"W1201 11:37:41.441567       1 clientconn.go:1223] grpc: addrConn.createTransport failed to connect to {https://127.0.0.1:2379  <nil> 0 <nil>}. Err :connection error: desc = \"transport: authentication handshake failed: remote error: tls: internal error\". Reconnecting...\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:37:42.445146 I | embed: rejected connection from \"127.0.0.1:59912\" (error \"open /run/config/pki/etcd/server.crt: no such file or directory\", ServerName \"\")\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"W1201 11:37:42.445412       1 clientconn.go:1223] grpc: addrConn.createTransport failed to connect to {https://127.0.0.1:2379  <nil> 0 <nil>}. Err :connection error: desc = \"transport: authentication handshake failed: remote error: tls: internal error\". Reconnecting...\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"W1201 11:37:43.949181       1 watcher.go:220] watch chan error: etcdserver: mvcc: required revision has been compacted\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:37:44.133888 I | embed: rejected connection from \"127.0.0.1:59932\" (error \"open /run/config/pki/etcd/server.crt: no such file or directory\", ServerName \"\")\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"W1201 11:37:44.133926       1 clientconn.go:1223] grpc: addrConn.createTransport failed to connect to {https://127.0.0.1:2379  <nil> 0 <nil>}. Err :connection error: desc = \"transport: authentication handshake failed: remote error: tls: internal error\". Reconnecting...\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:37:46.675657 I | embed: rejected connection from \"127.0.0.1:59952\" (error \"open /run/config/pki/etcd/server.crt: no such file or directory\", ServerName \"\")\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"W1201 11:37:46.676671       1 clientconn.go:1223] grpc: addrConn.createTransport failed to connect to {https://127.0.0.1:2379  <nil> 0 <nil>}. Err :connection error: desc = \"transport: authentication handshake failed: remote error: tls: internal error\". Reconnecting...\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:37:49.594583 I | etcdserver/api/etcdhttp: /health OK (status code 200)\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"W1201 11:37:51.484648       1 clientconn.go:1223] grpc: addrConn.createTransport failed to connect to {https://127.0.0.1:2379  <nil> 0 <nil>}. Err :connection error: desc = \"transport: authentication handshake failed: remote error: tls: internal error\". Reconnecting...\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:37:51.483810 I | embed: rejected connection from \"127.0.0.1:59988\" (error \"open /run/config/pki/etcd/server.crt: no such file or directory\", ServerName \"\")\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"W1201 11:37:52.339438       1 watcher.go:220] watch chan error: etcdserver: mvcc: required revision has been compacted\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"E1201 11:37:54.377819       1 dynamic_cafile_content.go:202] key failed with : open /run/config/pki/front-proxy-ca.crt: no such file or directory\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"E1201 11:37:54.377845       1 dynamic_cafile_content.go:202] key failed with : open /run/config/pki/ca.crt: no such file or directory\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:37:57.962079 I | embed: rejected connection from \"127.0.0.1:60042\" (error \"open /run/config/pki/etcd/server.crt: no such file or directory\", ServerName \"\")\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"W1201 11:37:57.963525       1 clientconn.go:1223] grpc: addrConn.createTransport failed to connect to {https://127.0.0.1:2379  <nil> 0 <nil>}. Err :connection error: desc = \"transport: authentication handshake failed: remote error: tls: internal error\". Reconnecting...\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"time=\"2021-12-01T11:37:58Z\" level=error msg=\"Port 30000 for service reactapp-service is already opened by another service\"\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:37:59.594571 I | etcdserver/api/etcdhttp: /health OK (status code 200)\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"E1201 11:38:02.851241       1 dynamic_serving_content.go:165] key failed with : open /run/config/pki/apiserver.crt: no such file or directory\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"E1201 11:38:02.851394       1 dynamic_cafile_content.go:202] key failed with : open /run/config/pki/front-proxy-ca.crt: no such file or directory\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"E1201 11:38:02.851411       1 dynamic_cafile_content.go:202] key failed with : open /run/config/pki/ca.crt: no such file or directory\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"E1201 11:38:02.860562       1 dynamic_serving_content.go:165] key failed with : open /run/config/pki/front-proxy-client.crt: no such file or directory\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"E1201 11:38:02.867217       1 dynamic_cafile_content.go:202] key failed with : open /run/config/pki/front-proxy-ca.crt: no such file or directory\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"E1201 11:38:02.867403       1 dynamic_cafile_content.go:202] key failed with : open /run/config/pki/ca.crt: no such file or directory\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"E1201 11:38:07.000297       1 dynamic_serving_content.go:165] key failed with : open /run/config/pki/ca.crt: no such file or directory\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"E1201 11:38:07.000553       1 dynamic_serving_content.go:165] key failed with : open /run/config/pki/ca.crt: no such file or directory\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"E1201 11:38:07.000297       1 dynamic_serving_content.go:165] key failed with : open /run/config/pki/ca.crt: no such file or directory\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"E1201 11:38:07.000772       1 dynamic_serving_content.go:165] key failed with : open /run/config/pki/ca.crt: no such file or directory\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:38:09.595572 I | etcdserver/api/etcdhttp: /health OK (status code 200)\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"E1201 11:38:15.501172       1 horizontal.go:227] failed to query scale subresource for Deployment/default/dj-app-deployment: deployments/scale.apps \"dj-app-deployment\" not found\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"I1201 11:38:15.501359       1 event.go:291] \"Event occurred\" object=\"default/dj-app-hpa\" kind=\"HorizontalPodAutoscaler\" apiVersion=\"autoscaling/v2beta2\" type=\"Warning\" reason=\"FailedGetScale\" message=\"deployments/scale.apps \\\"dj-app-deployment\\\" not found\"\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"E1201 11:38:15.514729       1 horizontal.go:227] failed to query scale subresource for Deployment/default/fl-app-deployment: deployments/scale.apps \"fl-app-deployment\" not found\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"I1201 11:38:15.515422       1 event.go:291] \"Event occurred\" object=\"default/fl-app-hpa\" kind=\"HorizontalPodAutoscaler\" apiVersion=\"autoscaling/v2beta2\" type=\"Warning\" reason=\"FailedGetScale\" message=\"deployments/scale.apps \\\"fl-app-deployment\\\" not found\"\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"W1201 11:38:18.986193       1 watcher.go:220] watch chan error: etcdserver: mvcc: required revision has been compacted\n","stream":"stderr"}
